lib/registries/addon/components/registries-states/actions-trigger/template.hbs

```diff
@@ -0,0 +1,19 @@
+<button class='btn btn-default' local-class='Trigger--default'>
+    {{#if (eq @state 'public')}}
```

Comment:
I think this would be a bit cleaner as a computed property or multiple computed properties.

```ts
get registrationState() {
    return {
        text: '...',
        icon: '....',
    };
} 
```

```hbs
<button>
    {{t this.registrationState.text}}
    {{fa-icon this.registrationState.icon}}
</button>
```